src/System.ServiceProcess.ServiceController/tests/ServiceBaseTests.cs

```diff
@@ -165,6 +170,7 @@ public void TestOnExecuteCustomCommand()
         public void TestOnContinueBeforePause()
         {
             _testService.Client.Connect(connectionTimeout);
+            Assert.Equal((int)PipeMessageByteCode.Connected, _testService.GetByte());
```

Comment:
These two lines always go together. That suggests it would make sense to extact into a method.
```c#
             _testService.Client.Connect(connectionTimeout);
+            Assert.Equal((int)PipeMessageByteCode.Connected, _testService.GetByte());
```